src/Estimators/tests/test_OneBodyDensityMatrices.cpp

```diff
@@ -70,6 +72,30 @@ class OneBodyDensityMatricesTests
     }
   }
 
+  /** Checking approximate equality for complex valued data as
+   *  two reals is not consistent with testing practices elsewhere in the code.
+   *  Values that are slightly off may now fall in approximation limit properly.
+   *
+   *  The smell from OperatorEstBase continuing the tradition of
+   *  packing everything into a Real buffer is not lost on me.
+   */
```

Comment:
ValueApprox doesn't work in this case?

Reply:
Once you get to the correct type yes, but I want to leave the explicit calls because I already sort into Real and Complex density matrix paths.  That will make it trivial to test both in the same build in the future.